cglims/api.py

```diff
@@ -11,6 +11,49 @@ def connect(config):
     api = ClinicalLims(config['host'], config['username'], config['password'])
     return api
 
+class ClinicalSample:
+
+    def __init__(self, sample):
+        """ Wrapper around the genologics Sample class
+
+        Args:
+            sample (genologics.Sample): the sample instance to extend
+        """
+        self.lims = sample
+        self._pipeline = None
+        self._apptag = None
```

Comment:
Mainly because I wanted to have lazy loading set up. I do realize in current set up this doesn't bring anything at all tho.